datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -1026,15 +1028,17 @@ mod tests {
         column_statistics: Vec<ParquetStatistics>,
     ) -> RowGroupMetaData {
         let mut columns = vec![];
+        let number_row = 1000;
```

Comment:
Before all unit test set each col with default `0 row`, will trigger num_rows == num_nulls

Reply:
I was wondering if this could cause problems in real files (for example, if the row counts were not included in the statistics that were written into the file).

However, I double checked the code and it seems like `ColumnChunkMetaData::num_values` is non nullable so I think we are good.